src/metrics/metrics.jl

```diff
@@ -42,11 +44,10 @@ Makes Metric types callable with arbitary arguments that are passed to associate
 """
 function (f::Metric)(raw, args...; kwargs...)::YAXArray
     axes::Tuple = (:timesteps, :species, :locations, :scenarios)[1:ndims(raw)]
-
-    return fill_axes_properties(f, f.func(DataCube(raw, axes), args...; kwargs...))
+    return fill_metadata!(@views(f.func(DataCube(raw, axes), args...; kwargs...)), f)
 end
 function (f::Metric)(rs::ResultSet, args...; kwargs...)::YAXArray
-    return fill_axes_properties(f, (f.func(rs, args...; kwargs...)))
+    return fill_metadata!(@views(f.func(rs, args...; kwargs...)), f)
```

Comment:
Huh does `@views()` do anything the way it's used here? 

I thought the macro required slice syntax (e.g., `x[:, 10:20]`) for it to actually produce a view:

```julia
x = rand(100, 100);

julia> typeof(@views(x[:, 10:20]))
SubArray{Float64, 2, Matrix{Float64}, Tuple{Base.Slice{Base.OneTo{Int64}}, UnitRange{Int64}}, true}

julia> typeof(@views(x))
Matrix{Float64} (alias for Array{Float64, 2})
```

Reply:
You're right, this doesn't make any sense. When I wrote this code I didn't understand `@views` very well. I'll change it back.